src/validation.cpp

```diff
@@ -4141,6 +4141,24 @@ void DumpMempool(void)
     }
 }
 
+//! Guess how far we are in the verification process at the given block index
+double GuessVerificationProgress(const ChainTxData& data, CBlockIndex *pindex) {
+    if (pindex == NULL)
+        return 0.0;
+
+    int64_t nNow = time(NULL);
+
+    double fTxTotal;
```

Comment:
Because there is a persistent "imitate the surrounding code" suggestion. If we actually want to migrate away from using that style, I'm all for it, but it means we should start enforcing a new style in reviews.